### DISCUSSION TITLE
Will 40 led settings cause issues on non 40 led setups?

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#319] [Q&A] [answered]
I was tinkering with the pack and noticed the 40 led cyclotron setting makes the nfilter puck light up red during normal operations. I really like it, although I'm on 36 led setup would it be a issue to keep it on 40?

**Comment by gpstar81** [comment, marked as answer]
It should be ok. You just will not have full n-filter lighting. Only 3 out of the 7 LEDs will light up during n-filter operation. There might also be a small hiccup in the cyclotron animation.

**Comment by Toy203** [comment]
Ah nice. Will see if it effects the venting look too much.
surprisingly enough I don't see any rotation issue so far...
